cmd/main.go

```diff
@@ -71,7 +71,6 @@ func BuildRootCommand(opt *Options) (*cobra.Command, error) {
 	if err := opt.bindCLIFlags(rootCmd.Flags()); err != nil {
 		return nil, err
 	}
-
```

Comment:
I think this should not be in the PR. I guess there's something missing from the PR?